File: atcoder/abs/5.swift
//usr/bin/env swift $0 $@;exit
let arr=readLine()!.characters.split{$0==" "}.map{Int(String($0))!}
let n=arr[0]
let a=arr[1]
let b=arr[2]
var r=0
for i in 1...n {
	var j=i
	var s=0
	while j>0 {
		s+=j%10
		j/=10
	}
	if a<=s&&s<=b {r+=i}
}
print(r)

File: atcoder/abs/5.pl
#!/usr/bin/perl
<>=~/ (.+) /;
for($i=1;$i<=$`;$i++){
	$s=0;
	$j=$i;
	for(;$j;$j=int($j/10)){$s+=$j%10}
	if($1<=$s&&$s<=$'){$r+=$i}
}
print$r
